crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1632,36 +1632,27 @@ impl IndexerStore for PgIndexerStore {
         let mutation_futures = object_mutation_chunks
             .into_iter()
             .map(|c| self.persist_object_mutation_chunk(c))
-            .collect::<Vec<_>>();
-        futures::future::join_all(mutation_futures)
-            .await
-            .into_iter()
-            .collect::<Result<Vec<_>, _>>()
-            .map_err(|e| {
-                IndexerError::PostgresWriteError(format!(
-                    "Failed to persist all object mutation chunks: {:?}",
-                    e
-                ))
-            })?;
+            .map(Either::Left);
         let deletion_futures = object_deletion_chunks
             .into_iter()
             .map(|c| self.persist_object_deletion_chunk(c))
-            .collect::<Vec<_>>();
-        futures::future::join_all(deletion_futures)
+            .map(Either::Right);
```

Comment:
I wanted to chain futures of two functions together previously and couldn't do it. Didn't think of using `Either`! TIL